TicketSwap/solar
# Input component - Version 2

# Description
We want to upgrade our inputs, to smoothen the upgrade we first created a V2.

## Changes

- [x] Add V2 input component

## How to test

- Checkout this branch
- `$ yarn storybook`
- See Components/Inputs/InputV2
